contrib/HACK.md

```diff
@@ -22,14 +22,20 @@ $ docker stack rm func
 $ docker network create func_functions --driver=overlay --attachable=true
 ```
 
-Now edit the `docker-compose.yml` file and add an attribute to the `functions` network of `attachable`.
+Now edit the `docker-compose.yml` file and replace the existing networks block with:
+
+```
+networks:
+    functions:
+        external:
+            name: func_functions
+```
 
 Now you can run the gateway as its own container and bind-mount in the HTML assets.
 
 ```
-$ cd faas
 $ docker run -v `pwd`/gateway/assets:/root/assets -v "/var/run/docker.sock:/var/run/docker.sock" \
--p 8080:8080 --network=func_functions -ti functions/gateway:latest-dev
+-p 8080:8080 --network=func_functions -itd functions/gateway:latest-dev
```

Comment:
What's "-itd" vs "-it" ?

Reply:
The majority of the time when working with faas i've run the gateway container in daemon mode so i thought i'd be nice for those copying and pasting these commands to run the gateway in the background

Reply:
Should it be "-d" though? I've never seen "-itd"?

Reply:
They can be chained `-i -t -d` is equivalent to `-itd` https://docs.docker.com/engine/reference/commandline/container_run/#options

Reply:
Oh sorry you mean, we don't need the `-it` flags, yes you're right, it should be just `-d`